modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_data.rb

```diff
@@ -27,15 +27,15 @@ def last_four_ssn
         end
 
         def birth_month
-          higher_level_review.birth_mm
+          higher_level_review.birth_date.month
```

Comment:
We're making assumptions about the structure of what the HLR returns from `#birth_date`... It wouldn't be that big of a deal to change if we need to in the future, but it's something to think about...